r/games/ChairRacingSimulator/imgUrlOptional.txt

```diff
@@ -0,0 +1 @@
+https://twitter.com/cSportDriving
```

Comment:
I think this should be https://twitter.com/BucketFullGames ? 😃

Reply:
True, thanks for watching! :)
I tend to trust what devs send me... ;)

Reply:
First thing I learned in IT/Ops...never trust devs! 😆